hardware_layer/hardware_arduino/src/Processing.cpp

```diff
@@ -0,0 +1,98 @@
+#include <ros/ros.h>
```

Comment:
This isn't part of hardware_sruino. Instead, put this in a different package named `control_layer`.